src/organizations.ts

```diff
@@ -0,0 +1,19 @@
+import {BaseApi} from './baseApi';
+import {PaginatedResponse} from './types';
+
+export interface Organization {
+    id: string;
+    name: string;
+    imageId: string;
+    locale?: string;
+    vertical?: 'Default' | 'Music';
+}
+
+/**
+ * API for working with Organizations
+ */
+export class OrganizationsApi extends BaseApi<PaginatedResponse<Organization>> {
+    getByUser(userId: string) {
```

Comment:
Seems silly, but maybe add some docs to each method?

Reply:
doc update pushed